include/spectrum.h

```diff
@@ -331,7 +331,13 @@ extern uint  __LIB__    zx_screenstr_callee(uchar row, uchar col) __smallc __z88
 #define zx_attr(a,b)              zx_attr_callee(a,b)
 #define zx_screenstr(a,b)         zx_screenstr_callee(a,b)
 
-#define zx_setcursorpos(a,b)      fputc_cons(22); fputc_cons(a); fputc_cons(b);
+// Sets the cursor position on screen
+// Remember to add 32 to both the desired row & col.
+#define zx_setcursorpos(row,col)      fputc_cons(22); fputc_cons(row); fputc_cons(col);
```

Comment:
These two macros seem a bit mismatched: one taking +32 and the other adding 32 - what's the thinking?